sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -109,11 +111,20 @@ def __str__(self):
         return str(self.message)
 
     def _build_message(self, body):
+        def _validate_body_type(body, allow_none=False):
+            if not (isinstance(body, (six.string_types, six.binary_type)) or (body is None and allow_none)):
+                raise TypeError("ServiceBusMessage body must be a string, bytes, None, or a list of string or bytes "
+                                "elements. Got instead: {}".format(type(body) if allow_none else "[... {} ...]"
+                                                                                                 .format(type(body))))
+
         if isinstance(body, list) and body:  # TODO: This only works for a list of bytes/strings
+            _validate_body_type(body[0])
             self.message = uamqp.Message(body[0], properties=self._amqp_properties, header=self._amqp_header)
             for more in body[1:]:
+                _validate_body_type(more)
                 self.message._body.append(more)  # pylint: disable=protected-access
```

Comment:
I tend to just remove the "if isinstance(body, list) and body" check part based on what I mentioned above

Reply:
++ I've reached out to Josh to see what we're normalizing on here.

Reply:
The answer is "don't take a list"; that may be a weirdness historical to our UAMQP.  This is a nice alignment with what the user studies found, that some folks got tripped up by this when they tried to send a list of messages in this fashion and "it worked" (but not as they expected)